app/views/users/new.html.erb

```diff
@@ -0,0 +1,46 @@
+<h1>Create a new User</h1>
+<!-- 
+-------------- Using casual HTML code --------------
+<form 
+  method="POST" 
+  action="/users" 
+  accept-charset="UTF-8">
+
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <div>
+    <label for="user_email">Email:</label>
+    <input id="user_email" name="user[email]" type="email" >
+  </div>
+  <div>
+    <label for="user_username">Username:</label>
+    <input id="user_username" name="user[username]" type="text" >
+  </div>
+  <div>
+    <label for="user_password">Password:</label>
+    <input id="user_password" name="user[password]" type="password" >
+  </div>
+  <input type="submit" value="Create a New User"/>
+</form>
+-->
+<!--
+-------------- Using Form_tag --------------
+<%= form_tag '/users', method:'post' do %>
+  <div>
+    <%= label_tag :email %>
+    <%= email_field_tag :email, nil, placeholder: 'something@gmail.com' %>
+  </div>
+
+  <div>
+    <%= label_tag :username %>
+    <%= text_field_tag :username %>
+  </div>
+
+  <div>
+    <%= label_tag :password %>
+    <%= password_field_tag :password %>
+  </div>
+
+  <%= submit_tag 'Create User' %>
+<% end %>
+ -->
+<%= render 'form' %>
```

Comment:
Nice !!

Reply:
thank you Mael Fosso for the review, the only reason why we left comments it was because of it was included in the requirements that's why we thought that it will be good if we left them on instead of removing them